hydra-node/test/Hydra/Chain/Direct/ContractSpec.hs

```diff
@@ -105,6 +106,15 @@ spec = describe "On-chain contracts" $ do
 -- Properties
 --
 
+prop_encode16BitsNaturalToCBOROnChain :: Property
+prop_encode16BitsNaturalToCBOROnChain =
+  forAll arbitrary $ \(word16 :: Word16) ->
+    let offChainCBOR = serialize' word16
+        onChainCBOR = fromBuiltin $ naturalToCBOR (fromIntegral word16)
+     in offChainCBOR == onChainCBOR
+          & counterexample ("offChainCBOR: " <> show offChainCBOR)
+          & counterexample ("onChainCBOR: " <> show onChainCBOR)
```

Comment:
This is a lot more detailed, but using `===` would've been sufficient for me here.

Reply:
Right, but we might have had a hard time telling which is which?